src/main/java/appeng/container/implementations/ContainerFormationPlane.java

```diff
@@ -38,7 +38,7 @@
 public class ContainerFormationPlane extends ContainerUpgradeable
 {
 
-	final PartFormationPlane storageBus;
+	private final PartFormationPlane storageBus;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "storageBus" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)